### DISCUSSION TITLE
Dialog with inputText causes close icon to wrap under Dialog header text

### DISCUSSION THREAD

**Discussion by chrishj59** [primefaces/community#391] [PrimeReact] [answered]
I would like to have input fields within a dialog so the user can update fields and also use a picklist.
I notice that if I use <InputText the close button wraps under the title text.

Using the example from the docs - if I add the inputText field the wrapping of the close icon occurs without the <inputText.. /> line the close icon appears on the right of the dialog as expected

     <Dialog
	      visible={showCategoryDlg}
	      header="Edit Brand Details"
	      modal
	      style={{ width: '50vw' }}
	      onHide={() => setShowCategoryDlg(false)}>
	     <p className="m-0">
			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
			culpa qui officia deserunt mollit anim id est laborum.
	     </p>
             <InputText id="title" value="some value" />
     </Dialog>
I have tried Google and searching this forum but did find anything with PrimeReact around this.

Is there a setting I am missing?

**Comment by mibpaz** [comment]
it's likely your css.
i've copied and pasted your snippet onto this sandbox below, only renaming the useState and changing "value" to "placeholder" to be able to use the input

https://codesandbox.io/s/dialog-test-zxnr5q

**Reply by chrishj59** [reply, marked as answer]
The issue was due to importing apollo v10 theme over v9 without migrating the code to NextJs app router

**Comment by chrishj59** [comment]
The issue was when I upgraded the apollo theme without migrating nextJS to app router. Now works fine
